src/interfaces/standings.interface.ts

```diff
@@ -0,0 +1,51 @@
+  import {Problem} from './problem.interface'
+  import {Member} from './member.interface'
+  
+
+  export interface Standings {
+    contest: Contest
+    problems: Problem[]
+    rows: Row[]
+  }
+  
+  export interface Contest {
+    id: number
+    name: string
+    type: string
+    phase: string
```

Comment:
Phase is also an enum with values: `BEFORE, CODING, PENDING_SYSTEM_TEST, SYSTEM_TEST, FINISHED`